db/migrate/20160927223306_save_source_template_to_clone_stage.rb

```diff
@@ -0,0 +1,5 @@
+class SaveSourceTemplateToCloneStage < ActiveRecord::Migration[5.0]
+  def change
+    add_column :stages, :template_stage_id, :integer
```

Comment:
This really is just to show the OPS person where the clone came from. As a sanity check. We'll show it in a couple of the view templates showing stages.